sx.el

```diff
@@ -426,19 +429,13 @@ Run after `sx-init--internal-hook'."
 This is used internally to set initial values for variables such
 as filters.")
 
-(defmacro sx-init-variable (variable value &optional setter)
-  "Set VARIABLE to VALUE using SETTER.
-SETTER should be a function of two arguments.  If SETTER is nil,
-`set' is used."
-  (eval
-   `(add-hook
-     'sx-init--internal-hook
-     (lambda ()
-       (,(or setter #'setq) ,variable ,value))))
-  nil)
+(defmacro sx-init-variable (variable)
+  "Set up VARIABLE to be loaded from disk on init."
+  `(add-hook 'sx-init--internal-hook
+             (lambda () (stash-load ,variable))))
```

Comment:
Given that it's a programmer's interface, I was hesitant to introduce functionality that wasn't *specifically* requested. Having gone through to use it, though, you're right – there are very few cases in which it isn't desired. A `no-load` flag to `defstash` would suffice.